routes/trading.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../middlewares/authenticate')
+const { trade } = require('../controllers/tradingController')
+const { newTrade } = require('../middlewares/validators/trading')
+
+/**
+ * @swagger
+ * /trade:
```

Comment:
Can you also modify the API schema?

Reply:
the one in `public/apiSchema.json` ?

Reply:
https://github.com/Real-Dev-Squad/website-backend/pull/222/commits/7de13d33fef4497c26f6921780a28ce426b18a20